Concrete/Magento2PlatformOrderDecorator.php

```diff
@@ -67,8 +80,8 @@ public function save()
 
     public function setStateAfterLog(OrderState $state)
     {
-       $stringState = $state->getState();
-       $this->platformOrder->setState($stringState);
+        $stringState = $state->getState();
```

Comment:
O github tá mostrando que foi adicionado um espaço em branco no início da linha. Verificar se isso aconteceu mesmo.